autopush/src/client.rs

```diff
@@ -600,36 +603,37 @@ where
         }
         let now = ms_since_epoch();
         let elapsed = (now - webpush.connected_at) / 1_000;
-        let (ua_result, metrics_os, metrics_browser) = parse_user_agent(&user_agent);
+        let ua_info = UserAgentInfo::from(user_agent.as_ref());
```

Comment:
Sorry one last thing, you shouldn't need to parse it again:

```suggestion
        let ua_info = webpush.ua_info.clone();
```

Reply:
Thanks! I realized that we don't need to pass the user_agent string anymore, so that's also been removed.